### PR TITLE
Update dev bundle Mongo to 3.2

### PR DESCRIPTION
**Pull request by tmeasday**
Using vanilla mongo from mongodb.com for https://github.com/meteor/meteor/issues/6957

### CODE REVIEW

**Comment by tmeasday** [comment]
@Fabs the unit tests passed: https://travis-ci.org/meteor/meteor/builds/131989107

I'll admit, I'm surprised!!

**Comment by tmeasday** [comment]
Failing selftests:

  - mongo: meteor mongo
  - mongo: meteor mongo in unicode dir
  - hot-code-push: css hot code push

**Comment by tmeasday** [comment]
Hah, turns out the test literally checked for "2.6"

File: tools/tests/mongo.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ var testMeteorMongo = function (appDir) {
   mongoRun.match('connecting to: 127.0.0.1');
   // Note: when mongo shell's input is not a tty, there is no prompt.
   mongoRun.write('db.version()\n');
-  mongoRun.match(/2\.6\.\d+/);
+  mongoRun.match(/3.2\.\d+/);
```
Comment (maxnowack) [inline tools/tests/mongo.js]: I there a reason, why the `.` isn't escaped?

Comment (tmeasday) [inline tools/tests/mongo.js]: Whoa, good pickup, thanks

**Pull request merged**